fdbserver/OldTLogServer_6_2.actor.cpp

```diff
@@ -2813,6 +2813,16 @@ ACTOR Future<Void> restorePersistentState(TLogData* self,
 		removed.push_back(errorOr(logData->removed));
 		logsByVersion.emplace_back(ver, id1);
 
+		TraceEvent ev("Role", recruited.id());
+		ev.detail("As", Role::TRANSACTION_LOG.roleName)
+		    .detail("Transition", "BeginWithEpochInfo")
+		    .detail("Origination", "Restored")
+		    .detail("OnWorker", self->workerID)
+		    .detail("RecoveryCount", std::to_string(logData->recoveryCount))
+		    .detail("SharedTLog", self->dbgid.shortString())
+		    .detail("TLogCodeVersion", "6.2");
+		ev.trackLatest(recruited.id().shortString() + ".Role");
```

Comment:
In order for the events to be logged in the same order they are written in code, you will either need to call `ev.log()` or have the event be destructed.

Reply:
I see. Thanks!